src/lib.rs

```diff
@@ -301,14 +310,13 @@ impl<'req> PixelsBuilder<'req> {
         assert!(height > 0);
 
         PixelsBuilder {
-            request_adapter_options: wgpu::RequestAdapterOptions {
-                compatible_surface: None,
-                power_preference: wgpu::PowerPreference::Default,
-            },
+            request_adapter_options: None,
             device_descriptor: wgpu::DeviceDescriptor::default(),
+            backend: wgpu::BackendBit::PRIMARY,
             width,
             height,
             pixel_aspect_ratio: 1.0,
+            enable_vsync: true,
```

Comment:
Maybe it's better to just pass around a `wgpu::PresentMode`... I imagine the `Fifo` mode can be useful in some situations!